source/_integrations/konnected.markdown

```diff
@@ -22,7 +22,7 @@ The integration currently supports the following device types in Home Assistant:
 - Switch: Actuate a siren, strobe, buzzer or relay module.
 - Sensor: Periodic measurements from DHT temperature/humidity sensors and DS18B20 temperature sensors.
 
-This integration uses the [`discovery`](/integrations/discovery) component, which must be enabled for device discovery to work. If you don't want to use discovery, set the _host_ and _port_ for each device in the description.  
+This integration uses the [`ssdp`](/integrations/ssdp) component, which must be enabled for device discovery to work. If you don't want to use discovery, set the _host_ and _port_ for each device in the description.  
```

Comment:
Does the user have to enable SSDP that themselves? (Sound a bit weird to me).
```suggestion
This integration uses the [SSDP](/integrations/ssdp) integration, which must be enabled for device discovery to work. If you don't want to use discovery, set the _host_ and _port_ for each device in the description.
```

Reply:
ssdp is part of the default config so a user would need to manually disable it to cause issues with discovery.

Reply:
Now I'm even confused more.
Your last response:

> ssdp is part of the default config so a user would need to manually disable it to cause issues with discovery

The documentation:

> This integration uses the [`ssdp`](/integrations/ssdp) component, which must be enabled for device discovery to work.

So should they enable or disable it? But not enable, because it is part of the default configuration? But have to disable it for issues? what issues? How can they disable it?

🤷‍♂

Reply:
The language is attempting to notify a user that the Konnected integration requires the SSDP component to be enabled for discovery features to work.  

The SSDP component is enabled in the default home assistant config as described [here](https://www.home-assistant.io/integrations/ssdp/).   

If for some reason a User disabled SSDP, or removed the default config, they would need to enable the SSDP component for discovery to work in the Konnected integration. 

SSDP disabled seems like an exception but highlighting the dependency is still valuable.  Perhaps there is a clearer way to phrase this information?

Reply:
I think it is fine to assume the default and just remove that sentence. There are many other things that cause discovery not to work at all (missing mDNS repeaters on VLANs, Wi-Fi isolations or broadcast blocking, just a bad router).


```
This integration automatically discovers your Konnected devices. If you don't want to use discovery, set the `host` and `port` for each device in the description.
```